bbmri/modules/gbn-setup.sh

```diff
@@ -6,6 +6,12 @@ if [ "${ENABLE_GBN}" == "true" ]; then
 
 	# Set required variables
 	GBN_BROKER_ID=broker.bbmri.de
```

Comment:
In case it's test env you set them twice, first for production, and then for test
do it in if-then-else instead